### ISSUE TITLE
File not found: OwnCloud

### ISSUE THREAD

**Issue by maitret** [DroidPHP/DroidPHP#81]
In:
https://cloud.openmailbox.org/index.php/s/97a796dd55679d77512b78217c90c3a2

**Comment by Shnoulle** [comment]
It's the link on http://droidphp.github.io/ then :+1:

**Comment by brnl** [comment]
Yeah that download button on the homepage links to https://cloud.openmailbox.org/index.php/s/97a796dd55679d77512b78217c90c3a2, which is not valid anymore.

But the project seems abandoned. Too bad (I guess, I didn't test it ;-) ).

**Comment by maitret** [comment]
Not works :( 

![screenshot_2016-07-23-01-17-25](https://cloud.githubusercontent.com/assets/674491/17076239/4504dd96-5073-11e6-9530-6cf56c94f0f9.png)
